docs/network-flow-visibility.md

```diff
@@ -101,6 +104,7 @@ with the Name and Namespace set to `flow-aggregator`. If you deploy the Flow Agg
 Service with a different Name and Namespace, then either use the appropriate DNS
 name or the Cluster IP of the Service. Please note that the default values for
 `flowPollInterval` and `flowExportFrequency` parameters are set to 5s and 12, respectively.
+TLS communication between flow exporter to flow aggregator is enabled by default. 
```

Comment:
"the Flow Exporter and the Flow Aggregator"

Reply:
Done. Thanks!